src/main/java/com/stuypulse/robot/commands/arm/routines/ArmReady.java

```diff
@@ -30,7 +30,14 @@ protected ArmTrajectory getTrajectory(ArmState src, ArmState dest) {
                 .addState(dest);
         }
 
-        return super.getTrajectory(src, dest);
+        double wristSafeAngle = 90; // src.getShoulderState().getCos() > 0 ? 120 : 60;
+
+        return new ArmTrajectory()
+            .addState(src.getShoulderDegrees(), wristSafeAngle)
+            .addState(dest.getShoulderDegrees(), wristSafeAngle)
```

Comment:
this affects auton. Have an if !auton or something analogous around this, at least for midwest